t/api/03-auth.t

```diff
@@ -69,6 +69,82 @@ my $ws = OpenQA::WebSockets->new;
 
 my $ret;
 
+subtest 'access limiting for non authenticated users' => sub() {
```

Comment:
why the '()' on the 'sub'?

In this case I guess helper sub's would be nicer to look at.